doc/api/stream.md

```diff
@@ -1086,6 +1086,16 @@ added: v11.4.0
 
 Is `true` if it is safe to call [`readable.read()`][stream-read].
 
+##### readable.readableEncoding
+<!-- YAML
+added: REPLACEME
+-->
+
+* {null|string}
+
+Getter for the property `encoding` of a given `Readable` stream. The `encoding`
+property can be set using the [`readable.setEncoding()`][] method.
+
```

Comment:
Is this something that is useful for the public? If so: what use case would that likely look like?

Reply:
Maybe see here: https://github.com/nodejs/node/issues/445#issuecomment-475176635 , or did I miss something ?